src/index.ts

```diff
@@ -52,6 +52,11 @@ export const completedLabRequestsTable = getAsyncLifecycle(
   options,
 );
 
+export const declinedLabRequestsTable = getAsyncLifecycle(
+  () => import('./lab-tabs/data-table-extensions/declined-lab-requests-table-extension'),
+  options,
+);
```

Comment:
```suggestion
import DeclinedLabRequstTable from './lab-tabs/data-table-extensions/declined-lab-requests-table-extension';
export const declinedLabRequestsTable = getSyncLifecycle(DeclinedLabRequstTable,options);
```

Reply:
I added this in line with how the other tabs have been exported